packages/create-invoice-form/src/lib/create-invoice-form.svelte

```diff
@@ -8,7 +8,7 @@
   // Utils
   import { calculateInvoiceTotals } from "@requestnetwork/shared-utils/invoiceTotals";
```

Comment:
![image](https://github.com/user-attachments/assets/f98650ce-17dc-46d3-a28a-1a19f2ed84cc)


@rodrigopavezi 

The current implementation assumes that we are only doing conversion requests; I think we should have a way to select normal requests and conversion requests.

Reply:
I think it's okay to assume conversion :+1: 

I think we should remove the "Select" and "Select a" prefixes.

Should be

- Invoice currency (labeling)
- Payment chain
- Settlement currency